acceptance: TestCompose failed [/cockroach/cockroach: no such file or directory]

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17240299?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17240299?buildTab=artifacts#/) on master @ [c104fb193997552d3181fc74db98480e7f7fe0d5](https://github.com/cockroachdb/cockroach/commits/c104fb193997552d3181fc74db98480e7f7fe0d5):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.72s)
        psql_1       |     --- FAIL: TestGSS/4 (0.80s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.70s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.57s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.61s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.60s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.61s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.63s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.62s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.63s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.75s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (41.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-42897

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17241550?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17241550?buildTab=artifacts#/) on master @ [c104fb193997552d3181fc74db98480e7f7fe0d5](https://github.com/cockroachdb/cockroach/commits/c104fb193997552d3181fc74db98480e7f7fe0d5):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.57s)
        psql_1       |     --- FAIL: TestGSS/4 (0.84s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.60s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.52s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.63s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.58s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.62s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.67s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.61s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.59s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.73s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        Aborting on container exit...
        
�[Kpsql_1 exited with code 1
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (41.82s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17241774?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17241774?buildTab=artifacts#/) on master @ [c104fb193997552d3181fc74db98480e7f7fe0d5](https://github.com/cockroachdb/cockroach/commits/c104fb193997552d3181fc74db98480e7f7fe0d5):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (7.37s)
        psql_1       |     --- FAIL: TestGSS/4 (0.85s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.61s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.55s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.63s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.61s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.61s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.61s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.62s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.73s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (1.36s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        Aborting on container exit...
        
�[Kpsql_1 exited with code 1
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (35.99s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17241991?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17241991?buildTab=artifacts#/) on master @ [16c9cfb90074dc6a56e71b2d889004dfba014264](https://github.com/cockroachdb/cockroach/commits/16c9cfb90074dc6a56e71b2d889004dfba014264):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.84s)
        psql_1       |     --- FAIL: TestGSS/4 (0.83s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.60s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.57s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.65s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.74s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.66s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.63s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.65s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.60s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.73s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (39.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17243127?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17243127?buildTab=artifacts#/) on master @ [45b47f90445fdb0e0c8e07df863c3bbe17daa491](https://github.com/cockroachdb/cockroach/commits/45b47f90445fdb0e0c8e07df863c3bbe17daa491):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.60s)
        psql_1       |     --- FAIL: TestGSS/4 (1.03s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.58s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.52s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.61s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.58s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.58s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.59s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.64s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.58s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.70s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (40.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17244394?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17244394?buildTab=artifacts#/) on master @ [45b47f90445fdb0e0c8e07df863c3bbe17daa491](https://github.com/cockroachdb/cockroach/commits/45b47f90445fdb0e0c8e07df863c3bbe17daa491):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.60s)
        psql_1       |     --- FAIL: TestGSS/4 (0.80s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.63s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.57s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.62s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.59s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.60s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.62s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.61s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.60s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.77s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (48.64s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17244883?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17244883?buildTab=artifacts#/) on master @ [45b47f90445fdb0e0c8e07df863c3bbe17daa491](https://github.com/cockroachdb/cockroach/commits/45b47f90445fdb0e0c8e07df863c3bbe17daa491):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (8.09s)
        psql_1       |     --- FAIL: TestGSS/4 (0.96s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.80s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.59s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.90s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.80s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.81s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.88s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.80s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.62s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.74s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (37.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245032?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245032?buildTab=artifacts#/) on master @ [45b47f90445fdb0e0c8e07df863c3bbe17daa491](https://github.com/cockroachdb/cockroach/commits/45b47f90445fdb0e0c8e07df863c3bbe17daa491):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.70s)
        psql_1       |     --- FAIL: TestGSS/4 (0.81s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.64s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.56s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.65s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.61s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.61s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.63s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.61s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.62s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.76s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (31.99s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245138?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245138?buildTab=artifacts#/) on master @ [23e8220c3c26cfcd103f4f3ddd53f300217f8f92](https://github.com/cockroachdb/cockroach/commits/23e8220c3c26cfcd103f4f3ddd53f300217f8f92):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.78s)
        psql_1       |     --- FAIL: TestGSS/4 (0.83s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.62s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.55s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.67s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.62s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.63s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.64s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.63s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.64s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.76s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (39.89s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245574?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245574?buildTab=artifacts#/) on master @ [046c7aa9354de2057a0f36b116e67b161c10af11](https://github.com/cockroachdb/cockroach/commits/046c7aa9354de2057a0f36b116e67b161c10af11):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.53s)
        psql_1       |     --- FAIL: TestGSS/4 (0.80s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.61s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.55s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.63s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.59s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.61s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.64s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.60s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.60s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.72s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (36.52s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

acceptance.TestCompose [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245864?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_Acceptance/17245864?buildTab=artifacts#/) on master @ [45b47f90445fdb0e0c8e07df863c3bbe17daa491](https://github.com/cockroachdb/cockroach/commits/45b47f90445fdb0e0c8e07df863c3bbe17daa491):


```
        psql_1       | Running test
        psql_1       | --- FAIL: TestGSS (6.73s)
        psql_1       |     --- FAIL: TestGSS/4 (0.79s)
        psql_1       |         --- FAIL: TestGSS/4/cockroach (0.01s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/5 (0.64s)
        psql_1       |         --- FAIL: TestGSS/5/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/6 (0.55s)
        psql_1       |         --- FAIL: TestGSS/6/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err no server.host_based_authentication.configuration entry, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/7 (0.63s)
        psql_1       |         --- FAIL: TestGSS/7/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/8 (0.62s)
        psql_1       |         --- FAIL: TestGSS/8/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err GSSAPI realm \(MY.EX\) didn't match any configured realm, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/9 (0.63s)
        psql_1       |         --- FAIL: TestGSS/9/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/10 (0.66s)
        psql_1       |         --- FAIL: TestGSS/10/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       |     --- FAIL: TestGSS/11 (0.63s)
        psql_1       |         --- FAIL: TestGSS/11/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/12 (0.65s)
        psql_1       |         --- FAIL: TestGSS/12/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err system identity "tester@my.ex" did not map to a database role, got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |     --- FAIL: TestGSS/13 (0.76s)
        psql_1       |         --- FAIL: TestGSS/13/cockroach (0.00s)
        psql_1       |             gss_test.go:180: expected err , got fork/exec /cockroach/cockroach: no such file or directory
        psql_1       |             gss_test.go:184: expected authentication_method=gss, got 
        psql_1       | FAIL
        
�[Kpsql_1 exited with code 1
        Aborting on container exit...
         Container gss_psql_1  Stopping
         Container gss_psql_1  Stopped
         Container gss_cockroach_1  Stopping
         Container gss_cockroach_1  Stopped
         Container gss_kdc_1  Stopping
         Container gss_kdc_1  Stopped
        
    compose_test.go:95: exit status 1
    --- FAIL: TestCompose/ComposeGSS (41.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCompose.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/acceptance/compose/gss/docker-compose-python.yml
+++ b/pkg/acceptance/compose/gss/docker-compose-python.yml
@@ -23,7 +23,7 @@ services:
       timeout: 10s
       retries: 25
   python:
-    image: us-east1-docker.pkg.dev/crl-ci-images/cockroach/acceptance-gss-python:20241004-141441
+    image: us-east1-docker.pkg.dev/crl-ci-images/cockroach/acceptance-gss-python:20241009-173356
     user: "${UID}:${GID}"
     depends_on:
       cockroach:

--- a/pkg/acceptance/compose/gss/docker-compose.yml
+++ b/pkg/acceptance/compose/gss/docker-compose.yml
@@ -24,7 +24,7 @@ services:
       timeout: 10s
       retries: 25
   psql:
-    image: us-east1-docker.pkg.dev/crl-ci-images/cockroach/acceptance-gss-psql:20241004-140930
+    image: us-east1-docker.pkg.dev/crl-ci-images/cockroach/acceptance-gss-psql:20241009-173040
     user: "${UID}:${GID}"
     depends_on:
       cockroach:

